library/Fission/Internal/Orphanage/MultipartForm.hs

```diff
@@ -15,9 +14,3 @@ instance HasSwagger api => HasSwagger (MultipartForm Mem (MultipartData Mem) :>
   toSwagger _ = Proxy @api
              |> toSwagger
              |> addConsumes ["multipart" // "form-data"]
-             |> addParam param
```

Comment:
Just curious why we removed the param here. Seems like this route isn't very useful without an uploaded file param